packages/webpack-cli/lib/webpack-cli.js

```diff
@@ -382,92 +398,153 @@ class WebpackCLI {
             {
                 name: 'hot',
                 alias: 'h',
-                type: [Boolean, String],
+                configs: [
+                    {
+                        type: 'string',
+                    },
+                ],
                 negative: true,
                 description: 'Enables Hot Module Replacement',
                 negatedDescription: 'Disables Hot Module Replacement.',
             },
             {
                 name: 'analyze',
-                type: Boolean,
+                configs: [
+                    {
+                        type: 'boolean',
+                    },
+                ],
                 multiple: false,
                 description: 'It invokes webpack-bundle-analyzer plugin to get bundle information.',
             },
             {
                 name: 'progress',
-                type: [Boolean, String],
+                configs: [
+                    {
+                        type: 'string',
+                    },
+                    {
+                        type: 'boolean',
+                    },
+                ],
                 description: 'Print compilation progress during build.',
             },
             {
                 name: 'prefetch',
-                type: String,
+                configs: [
+                    {
+                        type: 'string',
+                    },
+                ],
                 description: 'Prefetch this request.',
             },
 
             // Output options
             {
                 name: 'json',
-                type: [String, Boolean],
+                configs: [
+                    {
+                        type: 'string',
+                    },
+                    {
+                        type: 'boolean',
+                    },
+                ],
                 alias: 'j',
                 description: 'Prints result as JSON or store it in a file.',
             },
 
             // For webpack@4
             {
                 name: 'entry',
-                type: String,
+
```

Comment:
Missing `type` and `configs`

Reply:
added - https://github.com/webpack/webpack-cli/pull/2508/commits/623c17a5c353b59f524d327a30968cb2b5843de5